packages/auth/src/platform_browser/persistence/session_storage.test.ts

```diff
@@ -66,8 +66,8 @@ describe('platform_browser/persistence/session_storage', () => {
       afterEach(() => sinon.restore());
 
       it('should emit false if sessionStorage setItem throws', async () => {
-      sinon.stub(Storage.prototype, 'setItem').throws(new Error('nope'));
-      expect(await persistence._isAvailable()).to.be.false;
+        sinon.stub(Storage.prototype, 'setItem').throws(new Error('nope'));
```

Comment:
nit: revert since this is not related to the TOTP changes? This and the events.test.ts file.

Reply:
Is this done with auto formatting? If so, maybe create a separate formatting pr on main branch, then rebase totp onto main. This way we get the correct format + won't merge too many formatting changes back to main branch later.

Reply:
Yes, this was created through formatting. For now, I have removed these files from this PR